src/blocks/ImageTextList/Component.tsx

```diff
@@ -0,0 +1,72 @@
+import type { StaticImageData } from 'next/image'
+
+import RichText from '@/components/RichText'
+import React from 'react'
+import { cn } from 'src/utilities/cn'
+
+import type { ImageTextList as ImageTextListProps } from '@/payload-types'
+
+import { Media } from '@/components/Media'
+
+type Props = ImageTextListProps & {
+  className?: string
+  enableGutter?: boolean
+  imgClassName?: string
+  staticImage?: StaticImageData
+}
+
+export const ImageTextList: React.FC<Props> = (props) => {
```

Comment:
nit: while you had this PR open we decided to omit `React.FC` from the typing, so this would become

```ts
export const ImageTextList = (props: Props) => {
```